<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Address_Book_Window</class>
 <widget class="QWidget" name="Address_Book_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget_Address">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>321</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_name_motify">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_email_motify">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_name_add">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_email_add">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_name_motify">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改姓名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_email_motify">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改邮箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_add">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>200</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认增加联系人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>修改联系人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Delete">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>437</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Send">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>-7</x>
     <y>-9</y>
     <width>971</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Background5.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>Btn_Send</zorder>
  <zorder>Btn_Delete</zorder>
  <zorder>listWidget_Address</zorder>
  <zorder>Edit_name_motify</zorder>
  <zorder>Edit_email_motify</zorder>
  <zorder>Edit_name_add</zorder>
  <zorder>Edit_email_add</zorder>
  <zorder>Btn_name_motify</zorder>
  <zorder>Btn_email_motify</zorder>
  <zorder>Btn_add</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
